listenbrainz/webserver/static/js/src/LastFMImporter.tsx

```diff
@@ -196,16 +188,36 @@ export default class LastFmImporter extends React.Component<
         this.countReceived += payload.length;
         return payload;
       }
-      if (/^5/.test(response.status.toString())) {
-        return await retry(`Got ${response.status}`);
+      // ignore 40x errors
+      if (!/^4/.test(response.status.toString())) {
```

Comment:
this test is a little broader than what it used to be. From what I understand, `response.ok` covers 2xx codes. We used to trigger on 5xx and skip everything else. Now we skip on 4xx and trigger on everything else (which could maybe include 3xx? 
I guess there's no 6xx, so it's not really an issue, but it's a bit weird to see a check using `4` when really it's a handler for errors starting with `5`

Reply:
I adapted it to follow the description of one of the tests: "it should retry if there is any other error".
But that is indeed a different mechanism and possibly not what we want to do.
Let me take a closer look at the LFM API.